src/Sulu/Component/Content/Tests/Unit/Document/Subscriber/PHPCR/SuluNodeTest.php

```diff
@@ -34,12 +34,12 @@ public function provideDelegateData()
             ['isNew', true],
             ['isModified', true],
             ['isSame', false, [$this->prophesize(NodeInterface::class)->reveal()]],
-            ['accept', true, [$this->prophesize(ItemVisitorInterface::class)->reveal()]],
-            ['revert', true],
+            ['accept', null, [$this->prophesize(ItemVisitorInterface::class)->reveal()]],
```

Comment:
The unit tests where build on mocks and so we needed do change here the accepted result as the decoratedService result always is null.